src/metafile.c

```diff
@@ -1343,19 +1343,21 @@ gdip_get_metafile_from (void *pointer, GpMetafile **metafile, ImageSource source
 		goto error;
 
 	switch (mf->metafile_header.Type) {
-	case METAFILETYPE_WMFPLACEABLE:
-	case METAFILETYPE_WMF:
+	case MetafileTypeWmfPlaceable:
+	case MetafileTypeWmf:
 		mf->base.image_format = WMF;
 		/* note: mtSize is in WORD, mtSize contains the METAHEADER, mf->length is in bytes */
 		mf->length = mf->metafile_header.Header.Wmf.mtSize * 2 - sizeof (METAHEADER);
 		break;
-	case METAFILETYPE_EMF:
-	case METAFILETYPE_EMFPLUSONLY:
-	case METAFILETYPE_EMFPLUSDUAL:
+	case MetafileTypeEmf:
+	case MetafileTypeEmfPlusOnly:
+	case MetafileTypeEmfPlusDual:
 		mf->base.image_format = EMF;
 		mf->length = mf->metafile_header.Header.Emf.nBytes - mf->metafile_header.Header.Emf.nSize;
 		adjust_emf_headers = TRUE;
 		break;
+    default:
+        goto error;
```

Comment:
status needs to be set here otherwise it'll return Ok status from previous call to gdip_get_metafileheader_from

Reply:
indentation needs some love too 😄